smoke-tests/src/test/java/com/splunk/opentelemetry/helper/AbstractTestContainerManager.java

```diff
@@ -35,6 +35,7 @@
     environment.put("OTEL_BSP_SCHEDULE_DELAY", "10ms");
     environment.put(
         "OTEL_EXPORTER_JAEGER_ENDPOINT", "http://" + COLLECTOR_ALIAS + ":14268/api/traces");
```

Comment:
Do we need this property now?

Reply:
Hmm, nope - I'll move it to that one test that actually uses it.